### ISSUE TITLE
issues when linking to static libQtProtobufWellKnownTypes.a

### ISSUE THREAD

**Issue by gmabey** [semlanik/qtprotobuf#113]
I have a scenario where I've compiled Qt 5.12.8 statically on ubuntu18.04 as well as qtprotobuf pulled from the master branch.

My problem is that I get a linker error `undefined reference to google::protobuf::Timestamp::Timestamp(google::protobuf::Timestamp const&)` when attempting to link my executable and I'm trying to figure out whether it's really the fault of libQtProtobufWellKnownTypes like it looks (to me):

```
root@a8f8ea13d4cd:/tmp/ss/CPC# ll /usr/lib/x86_64-linux-gnu/libQtProtobufWellKnownTypes.a
-rw-r--r-- 1 root root 2164 May 22 16:30 /usr/lib/x86_64-linux-gnu/libQtProtobufWellKnownTypes.a
root@a8f8ea13d4cd:/tmp/ss/CPC# ll /usr/lib/x86_64-linux-gnu/libQtProtobuf.a              
-rw-r--r-- 1 root root 4085206 May 22 16:29 /usr/lib/x86_64-linux-gnu/libQtProtobuf.a
root@a8f8ea13d4cd:/tmp/ss/CPC# nm /usr/lib/x86_64-linux-gnu/libQtProtobufWellKnownTypes.a

dummy.cpp.o:

mocs_compilation.cpp.o:

```
In other words, the libQtProtobufWellKnownTypes.a that was created seems to be too small in its file size, and it seems to be practically empty with respect to symbols.

Am I missing something obvious here?

**Comment by semlanik** [comment]
@gmabey  could you please give me more background about options for cmake you used? I will try to reproduce it locally.

**Comment by semlanik** [comment]
Checked locally it seems linking issue. Probably I forgot whole archive at some place.
